packages/streams/pigpio-seatalk.js

```diff
@@ -23,7 +26,11 @@ const debug = require('debug')('signalk:streams:pigpio-seatalk')
 
 const cmd = `
 import pigpio, time, signal, sys
-gpio= 19 #define gpio where the seatalk1 (yellow wire) is connected
+
+if  sys.argv[1] == "undefined":
+        gpio = 4					                            #Default GPIO4 if not set
+else:
+        gpio = int(filter(str.isdigit, sys.argv[1])) 	#Ggpio, info as "GPIOnn", from GUI setup. Sensing the seatalk1 (yellow wire)
```

Comment:
If the `isdigit` filter fails what will happen?

I'd just remove the filter and let things fail if it is not a number, as the arguments are coming from within the same "application" programmatically. We can also fix the js side if needed.

Reply:
..... little confused .... since the GUI stores the pin choice as a string, for example  "GPIO04" not as 4 ?? According to earlier discussion I thought this was by purpose, but now I'm lost...

Reply:
Oh right, I forgot that the value has the GPIO characters. Never mind.